components/rgb_led/include/rgb_led.h

```diff
@@ -33,6 +33,12 @@
 #define NOTIFICATION_RGB_GPIO   23
 #define RGB_LED_KEYBOARD_NUMBER 16
 #define RGB_LED_NOTIFICATION_NUMBER 2
+// The maximum brightness that the LEDs can be set to using the hsv2rgb function
+#define MAX_BRIGHTNESS 16
```

Comment:
I have flashed my UART controller to request a full 500ma, making this a very acceptable brightness level, without being overpowering, but the exact value might need to be changed since most board only do 100ma and might make this much dimmer.